be/src/runtime/runtime_state.cpp

```diff
@@ -511,4 +512,8 @@ Status RuntimeState::reset_epoch() {
     return Status::OK();
 }
 
+bool RuntimeState::is_jit_enabled() const {
+    return _query_options.__isset.enable_jit && _query_options.enable_jit && JITEngine::get_instance()->support_jit();
```

Comment:
should we put `JITEngine::get_instance()->support_jit();` to the first?  seems it won't change since be start

Reply:
will chagne in next PR.